src/components/CommentReplyPreview/CommentReplyPreview.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import { CommentReplyPreview } from "./CommentReplyPreview";
+import { CommentType } from "../../types";
+
+export default { title: "CommentReplyPreview" };
+
+const replyComment: CommentType = {
+  id: 25487686,
+  body:
+    "Beau Jos pizza in Idaho Springs is a great place for mountain pizza pies. Order one with extra thick crust and drizzle it with honey. Y'all can try the Challenge if you fancy, and sketch on your napkins so your art can join their walls. This was 15 years ago, but I hope it's still there! As for music, anything from Boulder's own Big Head Todd &amp; the Monsters - 'Broken Hearted Savior' is a good start, with 'Bittersweet' a good road track. I'm jealous!!!",
+  date: "26 July 2013 4:13pm",
+  isoDateTime: "2013-07-26T15:13:20Z",
+  status: "visible",
+  webUrl: "https://discussion.theguardian.com/comment-permalink/25487686",
+  apiUrl: "https://discussion.guardianapis.com/discussion-api/comment/25487686",
+  numRecommends: 0,
+  isHighlighted: false,
+  userProfile: {
+    userId: "2762428",
+    displayName: "FrankDeFord",
+    webUrl: "https://profile.theguardian.com/user/id/2762428",
+    apiUrl:
+      "https://discussion.guardianapis.com/discussion-api/profile/2762428",
+    avatar: "https://avatar.guim.co.uk/user/2762428",
+    secureAvatarUrl: "https://avatar.guim.co.uk/user/2762428",
+    badge: []
+  },
+  responses: [],
+  metaData: {
+    commentCount: 2,
+    staffCommenterCount: 1,
+    editorsPickCount: 0,
+    blockedCount: 0,
+    responseCount: 1
+  }
+};
+
+export const Default = () => (
+  <CommentReplyPreview replyComment={replyComment} pillar={"sport"} />
```

Comment:
These props look wrong, I think that should be `commentBeingRepliedTo`, not `replyComment`. Stuff like will be caught by linting in future once I complete a PR I'm working on!